utils/s3/client.cc

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include <rapidxml.h>
+#include <seastar/core/coroutine.hh>
```

Comment:
rapidxml must be added to the dependency list

Reply:
From the website:

**Last Updated**
2013-04-26

While that's my birthday, maybe we should look for something more recent. On the other hand, if it works, it works.

Reply:
No incidents reported since 2013 :sunglasses: